src/dmd/frontend.d

```diff
@@ -14,10 +14,18 @@ module dmd.frontend;
 
 // Online documentation: https://dlang.org/phobos/dmd_frontend.html
 
-/**
-Initializes the DMD compiler
+import dmd.dmodule : Module;
+
+shared static this()
+{
+    initDMD();
+}
```

Comment:
I'm not sure if this is the best idea if we want to allow multiple instances of the compiler in the future.